CHANGELOG.rst

```diff
@@ -1,10 +1,8 @@
 Changelog
 =========
 
-1.7 - `master`_
-~~~~~~~~~~~~~~~
-
-.. note:: This version is not yet released and is under active development.
+1.7 - 2016-12-11
```

Comment:
Sigh, I even thought of this and looked at a calendar and managed to screw it up still.